src/natcap/invest/hra.py

```diff
@@ -2606,7 +2607,7 @@ def _get_criteria_dataframe(base_criteria_table_path):
 
     # Validate the column header, which should have 'criteria type'
     criteria_df.columns = [
-        x if isinstance(x, str) else None for x in
+        x.strip() if isinstance(x, str) else None for x in

```

Comment:
Wouldn't this also be handled by the `strip` kwarg?

Reply:
`criteria_df` and `info_df` (above) are still created from `pandas.read_csv()` here, not `utils.read_csv_to_dataframe()`. This is partly because of the need to handle possible excel files in this one model, and partly because they are being converted to uppercase instead of lowercase. The data is indexed throughout the model using all-uppercase titles, which I'd be in favor of replacing with lowercase, unless there's a reason for it to be uppercase..

Reply:
Ah, that's right ... the excel/CSV compatibility will make things a little more interesting.

As for whether the CSV headers are all-uppercase or all-lowercase, I don't personally have a preference and it sounds like an implementation decision made by the engineer who wrote this version of the model (who has since moved on).  Consistency is nice to have, so I'd support converting these to be lowercase in HRA.

Reply:
After taking a poll on slack, it looks like lowercase is preferred by all, so I'll go ahead with that.

Reply:
Is there any reason that a column name wouldn't be a `str`?

Reply:
I can't think of one, but I suppose a user might provide a numeric column name or something, which would probably be a good case to test against in our test suite.